wgpu-hal/src/gles/queue.rs

```diff
@@ -452,9 +452,16 @@ impl super::Queue {
                         _ => unreachable!(),
                     }
                 } else {
-                    let bytes_per_image =
-                        copy.buffer_layout.rows_per_image.map_or(1, |rpi| rpi.get())
-                            * copy.buffer_layout.bytes_per_row.map_or(1, |bpr| bpr.get());
+                    let bytes_per_row = copy.buffer_layout.bytes_per_row.map_or(
+                        copy.size.width / format_info.block_dimensions.0 as u32,
```

Comment:
this seems wrong? the equation here would return the block count per row, not bytes per row

Reply:
Indeed